src/plugins/intel_gpu/src/plugin/ops/fake_convert.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "intel_gpu/plugin/program_builder.hpp"
+#include "intel_gpu/plugin/common_utils.hpp"
+
+#include "openvino/op/fake_convert.hpp"
+
+#include "intel_gpu/primitives/fake_convert.hpp"
+
+namespace ov {
+namespace intel_gpu {
+
+static void CreateFakeConvertOp(ProgramBuilder& p, const std::shared_ptr<ov::Node>& op) {
+    validate_inputs_count(op, {2, 3});
+    const auto inputs = p.GetInputInfo(op);
+    const std::string layerName = layer_type_name_ID(op);
+    ov::element::Type destination_type = ov::element::Type_t::f8e4m3;
+    if (auto fake_convert_v13 = std::dynamic_pointer_cast<ov::op::v13::FakeConvert>(op)) {
+        destination_type = fake_convert_v13->get_destination_element_type();
+    } else {
+        OPENVINO_THROW("[GPU] Can't cast FakeConvert operation to any supported version");
+    }
```

Comment:
This is not needed, `REGISTER_FACTORY_IMPL(v13, FakeConvert)` does the job for you. Just update the type of the op argument to `const std::shared_ptr<ov::op::v13::FakeConvert>& op` for consistency

Reply:
Fixed it.